bld/unit_testers/build-namelist_test.pl

```diff
@@ -138,7 +138,7 @@ sub make_config_cache {
 #
 # Figure out number of tests that will run
 #
-my $ntests = 914;
+my $ntests = 924;
```

Comment:
@ekluzek I will try it; however, I disagree that the "toosmalls" may not be used with n_dom_pfts and n_dom_landunits. Here's an example: a user may wish to run with 2 dominant pfts and 2 dominant landunits and may still want to specify that a particular landunit shall be used if greater than 10%. As written, the code can do that.

> @slevisconsulting I pushed a change to the build-namelist tester to check that the toosmalls weren't being used with n_dom_landunit. It's currently doing the wrong thing there, so it needs some work. But, you can run the tester to show that things are OK. Other's have complained that it's hard to work with, so I can help you with it. I think just using it shouldn't be a problem though.

Reply:
@ekluzek I will wait to hear from you on this before I `git pull` your changes.

Reply:
@ekluzek does it work if I `git commit` and `git push` my revisions so that you may take a look, and then later I `git pull` your changes? Or will that order mess things up?

Reply:
The commits and push that I did just changed bld/unit_testers/build-namelist_test.pl, so it won't interfere with anything you are doing outside of that. You will need to pull those changes before you push anything else though.

Reply:
Ran `./build-namelist_test.pl`
and got
`Successfully ran all testing for build-namelist`
so I think we're good, other than still examining whether the `toosmall_*` parameters should coexist with the `n_dom_*` parameters. I vote YES, as explained earlier in this thread.